uaclient/security_status.py

```diff
@@ -192,15 +193,15 @@ def filter_security_updates(
 
 def get_ua_info(cfg: UAConfig) -> Dict[str, Any]:
     """Returns the Pro information based on the config object."""
+    is_attached = _is_attached(cfg).is_attached
     ua_info = {
-        "attached": False,
+        "attached": is_attached,
         "enabled_services": [],
         "entitled_services": [],
     }  # type: Dict[str, Any]
 
-    status_dict = status(cfg=cfg, show_all=True)
-    if status_dict["attached"]:
-        ua_info["attached"] = True
+    if is_attached:
+        status_dict = status(cfg=cfg, show_all=True)
```

Comment:
This is already a great improvement and I think it would be good to merge this.
I think we could go a step further and directly use `ESMAppsEntitlement` and `ESMAppsEntitlement` here instead of `status`. One downside of `status` is that it gets the status of all services, which can include, for example, talking to the livepatch api (though the response is cached).
But if you think that's too much for this PR, then we can do it at a later time.

Reply:
not too much - I think it makes sense to be done here right away